### ISSUE TITLE
Out of date?

### ISSUE THREAD

**Issue by Johnlon** [cucumber/cucumber-json-schema#6]
This repo seems recently modified but is this actually up to date.

What about embeddings and before and after blocks and so on?

Seems only to be a base minimum of it here.

Unfortunately almost all JSON Vs schema validators won't complain if the JSON contains a load of stuff not declared by the schema so without such a strict be validator then one will have to go through the cuke code by hand to correlate the schema here with the actual product .

Am I missing something.

I was hoping this was complete as I wanted to work on an extension but I feel now I need to read the cuke code to understand the file format.

Thanks

**Comment by aslakhellesoy** [comment]
Yes, this is currently very out of date. But might be updated soon I hope!

**Comment by Johnlon** [comment]
@aslakhellesoy 
Btw please also mediate on this other issue as it was treated unreasonably IMHO...

https://github.com/cucumber/cucumber-jvm/issues/2495

**Comment by Johnlon** [comment]
Hi Did it ever get updated?
